<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_mitarb_einzeln</class>
 <widget class="QDialog" name="Dialog_mitarb_einzeln">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Vorname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_nr"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Personalnummer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nachname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ID Scanner</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_vona"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_nana"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_idscan"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nr</tabstop>
  <tabstop>lineEdit_vona</tabstop>
  <tabstop>lineEdit_nana</tabstop>
  <tabstop>lineEdit_idscan</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
